onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -495,6 +559,20 @@ def ortvalue_from_shape_and_type(shape=None, element_type=None, device_type='cpu
         return OrtValue(C.OrtValue.ortvalue_from_shape_and_type(shape, element_type,
                         C.OrtDevice(get_ort_device_type(device_type), C.OrtDevice.default_memory(), device_id)))
 
+    @staticmethod
+    def ort_value_from_sparse_tensor(sparse_tensor):
+        '''
+        The function will construct an OrtValue instance from a valid SparseTensor
+        The new instance of OrtValue will assume the ownership of sparse_tensor
+        '''
+        return OrtValue(C.OrtValue.ort_value_from_sparse_tensor(sparse_tensor._get_c_tensor()))
+
+    def as_sparse_tensor(self):
+        '''
+        The function will return SparseTensor contained in this OrtValue
+        '''
+        return SparseTensor(self._ortvalue.as_sparse_tensor())
```

Comment:
Is the underlying SparseTensor still held by the OrtValue ?

Reply:
By a copy of OrtValue